Documentation/ceph-block.md

```diff
@@ -60,9 +60,9 @@ parameters:
     imageFeatures: layering
 
     # The secrets contain Ceph admin credentials.
-    csi.storage.k8s.io/provisioner-secret-name: rook-ceph-csi
+    csi.storage.k8s.io/provisioner-secret-name: rook-csi-rbd-provisioner
     csi.storage.k8s.io/provisioner-secret-namespace: rook-ceph
-    csi.storage.k8s.io/node-stage-secret-name: rook-ceph-csi
+    csi.storage.k8s.io/node-stage-secret-name: rook-csi-rbd-node-stage
```

Comment:
Can we drop the "stage" suffix on the secret name? `rook-csi-rbd-node` and `rook-csi-rbd-provisioner` sound good.

Reply:
I blindly followed the CSI team suggestions here, so @Madhu-1 @ajarr what do you think?

Reply:
agree we can drop stage from name

Reply:
ok